worksheet_A/TerminalHacking.cpp

```diff
@@ -4,8 +4,27 @@
 #include "stdafx.h"
 #include "WordList.h"
 
+using namespace std;
 const int wordLength = 5;
 const int numberOfWords = 15;
+int playerLives = 4; 
+std::string guessWord;
+int wordLengthLetter;
+int simScore = 0;
```

Comment:
Do these need to be global variables? Probably better to declare them inside `main()`.